components/nifosg/nifloader.cpp

```diff
@@ -811,7 +811,12 @@ namespace NifOsg
                 }
                 else if (affectors->recType == Nif::RC_NiParticleRotation)
                 {
-                    // unused
```

Comment:
Would really be nice to get a verification of this before we suddenly start to do something vanilla doesn't.

Reply:
noone anwers me on discord....

Reply:
@EquiNox seems to say rotations aren't part of vanilla...